man/systemd-sysv-generator.xml

```diff
@@ -60,6 +60,20 @@
     <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Environment</title>
+
+    <para>Special variables understood by <command>systemd-sysv-generator</command>:</para>
+
+    <variablelist class='environment-variables'>
+      <varlistentry>
+        <term><varname>$SYSTEMD_SYSVINIT_PATH</varname></term>
+
+        <listitem><para>Controls where systemd looks for SysV init scripts.</para></listitem>
+      </varlistentry>
+    </variablelist>
+  </refsect1>
```

Comment:
might be stuff to move to docs/ENVIRONMENT.md. It sounds like this would not be useful outside of a hacker/developer/debugger world